middleware/request_id.go

```diff
@@ -13,8 +13,13 @@ import (
 	"golang.org/x/net/context"
 )
 
-// RequestIDHeader is the name of the header used to transmit the request ID.
-const RequestIDHeader = "X-Request-Id"
+const (
+	// RequestIDHeader is the name of the header used to transmit the request ID.
+	RequestIDHeader = "X-Request-Id"
+
+	// DefaultRequestIDLengthLimit is the default maximum length for header value.
```

Comment:
Seems like you're missing a "the" here - maybe specify that this only applies to the request ID header too (should be obvious given the context but I guess it's better to be explicit): "for the request ID header value"